src/main/java/com/algorand/algosdk/v2/client/model/Account.java

```diff
@@ -7,7 +7,7 @@
 
 import com.algorand.algosdk.crypto.Address;
 import com.algorand.algosdk.v2.client.common.PathResponse;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.*;
```

Comment:
it is considered a poor practice in Java to import '*'. Why did it happen at all?

Reply:
Linters actually may complain on both. Does it make the generator code more tricky?

Reply:
I checked JavaGenerator code.
Looks like `Tools.addImport(imports, "com.fasterxml.jackson.annotation. JsonIgnore")` can be moved into `getBase64EncodedArray` and to be conditional if needed since this code in `close()`:
```
        JavaGenerator.append(modelFileWriter, Tools.getImports(imports));
        JavaGenerator.append(modelFileWriter, currentModelBuffer);
```

Reply:
great! I will go to change the code accordingly

Reply:
It looks to me that it is working! I will soon have a small PR to resolve the wildcard-import in `generator`, would you like to take a pass?